js/gulpfile.js

```diff
@@ -56,13 +56,13 @@ knownTargets.forEach((target) => {
     ));
 });
 
-// The main "apache-arrow" module builds the es5/umd, esnext/cjs,
+// The main "apache-arrow" module builds the es2015/umd, esnext/cjs,
 // esnext/esm, and esnext/umd targets, then copies and renames the
 // compiled output into the apache-arrow folder
 gulp.task(`build:${npmPkgName}`,
     gulp.series(
         gulp.parallel(
-            `build:${taskName(`es5`, `umd`)}`,
+            `build:${taskName(`es2015`, `umd`)}`,
```

Comment:
@trxcllnt I was hoping this would be the main thing to do but the bundle isn't right.

Why is there no es2015 bundle? Why is there an externs file for es5?

<img width="368" alt="Screen Shot 2021-04-12 at 09 48 01" src="https://user-images.githubusercontent.com/589034/114431287-2d182200-9b74-11eb-8b8c-341e7cff8d94.png">